media/js/base/mozilla-monitor-button.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// create namespace
+if (typeof window.Mozilla === 'undefined') {
+    window.Mozilla = {};
+}
+
+Mozilla.MonitorButton = (function(Mozilla) {
+    'use strict';
+
+    var monitorButton = document.getElementById('fxa-monitor-submit');
+    var buttonURL = monitorButton.getAttribute('href');
+
+    // fetch request
```

Comment:
@alexgibson howdy! -would love your thoughts on this fetch request, any gaps?

Reply:
@amychurchwell tested this locally and it seems to be doing the right thing to me 👍 

I'd suggest putting this up on demo so the FxA / Monitor folks can help vet the full flow?

Reply:
@groovecoder hi Luke! I threw up the current WIP page on demo so we can test the Monitor functionality. https://www-demo2.allizom.org/en-US/firefox/67.0.5/whatsnew/

It seems to be working locally, but I could not seem to get  the [fxa-dev-launcher](https://github.com/vladikoff/fxa-dev-launcher#basic-usage-example-in-os-x) to work for me. I tried `FXA_ENV=stage npm start`

Reply:
Sorry I missed this until now. Is this still going out as a WNP for 67.0.5? I got the 67 update and was sent here: https://www.mozilla.org/en-US/firefox/67.0/whatsnew/?oldversion=66.0.5

Reply:
And https://www-demo2.allizom.org/en-US/firefox/67.0.5/whatsnew/ doesn't seem to have anything more than a Monitor link?

Reply:
@groovecoder This will be going out for 67.0.5, so the 67 update is expected behavior. 👍 
Do you have time to do a zoom call today? Should be more than a Monitor link, but I have been not able to replicate the behavior on local on demo and I think it's because my setup is not correct? (fxa-dev-launcher)